dijet/selection/dijet.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+
+from typing import Tuple
+from columnflow.util import maybe_import
+from columnflow.selection import Selector, SelectionResult, selector
+
+ak = maybe_import("awkward")
+
+
+def masked_sorted_indices(mask: ak.Array, sort_var: ak.Array, ascending: bool = False) -> ak.Array:
+    """
+    Helper function to obtain the correct indices of an object mask
+    """
+    indices = ak.argsort(sort_var, axis=-1, ascending=ascending)
+    return indices[mask[indices]]
```

Comment:
Since this is quite a useful generic function that could be used in several places, you might consider putting it in its own module, e.g. `util.py`.